### DISCUSSION TITLE
How to cancel a channel opening, wich tx is not shown

### DISCUSSION THREAD

**Discussion by Steampowered-hippo** [lightningnetwork/lnd#8727] [Troubleshooting]
Hi,
I am using start9 with LND 0.17.4.
I tried to open a channel and checked "spend unconfirmed output".
Unfortunately the tx of the unspend output has a very low fee and seems to never get confirmed. In this state, the tx of my channel opening is not visible in the mempool. LND seems to broadcast the tx on and on.
I can not find a possibility to cancel this tx. Is there any?

**Comment by bitromortac** [comment]
You would need to double spend the output that's used for channel opening. Is the transaction you try to spend part of the LND wallet?

**Reply by Steampowered-hippo** [reply]
I tried to use OnChain funds for channel opening. A part of these funds were unconfirmed.
How does your solution (double spend) looks like? How to do so?

**Reply by bitromortac** [reply]
It is difficult to give precise advice without knowing the exact transactions and to which wallet they belong to. I think it would be easier to go through the process on slack, if you could join there. Otherwise the broader idea would be to either double spend the input of the unconfirmed transaction or to double spend the channel funding input such that the channel can never confirm, but this depends also on whether the unconfirmed transaction is already in the mempool or not. Would you be able to share transaction ids? You can find out more about the outputs via `lncli listchaintxns` and `lncli listunspent`, looking for the transaction ids.

**Reply by Steampowered-hippo** [reply]
Can I anyhow use the commandline commands, even if I am on a start9 server?

**Reply by bitromortac** [reply]
I'm not familiar with start9, you would need access to the command line (perhaps https://docs.start9.com/0.3.5.x/user-manual/ssh is helpful).

**Reply by Steampowered-hippo** [reply]
unfortunately I receive an rpc-error, when I try to fire your commands. (logged in via ssh, into the docker, LND up and running).

**Reply by bitromortac** [reply]
Could you join [Slack](https://lightning.engineering/slack.html), I think it will be easier to troubleshoot there (could send me a message)?

**Reply by Steampowered-hippo** [reply]
Ok, I joined slack with your link.
which Channel?
